ui/app/routes/vault/cluster/auth.js

```diff
@@ -7,27 +7,35 @@ import { service } from '@ember/service';
 import ClusterRouteBase from './cluster-route-base';
 import config from 'vault/config/environment';
 
-export default ClusterRouteBase.extend({
-  queryParams: {
-    authMethod: {
-      replace: true,
-    },
-  },
-  flashMessages: service(),
-  version: service(),
+export default class AuthRoute extends ClusterRouteBase {
+  queryParams = {
+    wrapped_token: { refreshModel: true },
```

Comment:
don't see a reason to rename as `wrappedToken` I think it's easier to refer consistently as 'wrapped_token'?